lib/paperclip/has_attached_file.rb

```diff
@@ -100,6 +101,20 @@ def add_active_record_callbacks
       end
     end
 
+    def add_background_processing
+      require "paperclip/process_job"
```

Comment:
I added require in this place, to not force `active_job` presence for people that won't use this feature